rundeckapp/src/test/groovy/rundeck/controllers/ApiControllerSpec.groovy

```diff
@@ -303,6 +310,44 @@ class ApiControllerSpec extends Specification implements ControllerUnitTest<ApiC
         ]
     }
 
+    @Unroll
+    def "api system info"() {
```

Comment:
Thanks for adding a test case for the feature. It could be much better to have a more informative test case name with the intention.